pkgs/development/tcl-modules/by-name/ji/jimlib/package.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  fetchFromGitLab,
+  mkTclDerivation,
+}:
+
+mkTclDerivation rec {
+  pname = "jimlib";
+  version = "0.15.0-unstable-2024-10-31";
+
+  src = fetchFromGitLab {
+    owner = "dbohdan";
+    repo = "jimlib";
+    rev = "6a50339a6fbfc26d793b987c939cdd82dc1db490";
+    sha256 = "sha256-EkYjaQqH9I8SCS424cpsyfigPweLLLLPvRkO+vHHCJ8=";
+  };
+
+  dontBuild = true;
+  doCheck = true;
+
+  checkPhase = ''
+    tclsh ./jimlib.tcl
+  '';
+
+  installPhase = ''
+    mkdir -p $out/lib/jimlib
+    cp $src/jimlib.tcl $out/lib/jimlib/jimlib.tcl
+    cp $src/pkgIndex.tcl $out/lib/jimlib/pkgIndex.tcl
+  '';
+
+  meta = with lib; {
+    homepage = "https://wiki.tcl-lang.org/page/jimlib";
+    description = "Jimlib is a personal standard library for Jim Tcl.";
+    license = licenses.mit;
```

Comment:
From the readme:

> MIT, Tcl, and BSD.
See the top of jimlib.tcl.

Reply:
Updated the license block. The line mentioning BSD ](https://gitlab.com/dbohdan/jimlib/-/blob/master/jimlib.tcl#L77 ) refers to code reused from tcllib, which is listed as Tcl licensed on nixpkgs.

Doesn't seem like there's a full license text included for BSD in either jimlib or upstream tcllib so I omitted it as I can't determine which version it was originally referring to

Reply:
According to dbohdan it's bsd3

Reply:
Makes sense, thank you for checking. Updated